lib/susedistribution.pm

```diff
@@ -114,6 +114,28 @@ sub init_cmd {
     ## keyboard cmd vars end
 }
 
+
+sub init_desktop_runner {
+    my ($timeout, $program) = @_;
```

Comment:
Well, I would have made timeout the second parameter so that you could leave it undefined but ok

Reply:
It's a good point, because we don't always set timeout, so it can break tests, I'll submit PR to fix that

Reply:
Please, review this one https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4389 and please merge needles.